draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -176,7 +176,7 @@ defined in [@!RFC1982].  Otherwise, catalog consumers might not notice
 updates to the catalog zone's contents.
 
 There is no requirement to be able to query the catalog zone via recursive nameservers.
-Catalog consumers SHOULD ignore NS record at apex.
+Catalog consumers MUST ignore NS record at apex.
```

Comment:
```suggestion
Catalog consumers MUST ignore the absence of NS record at apex.
```

Reply:
(although I suspect BIND might currently not accept that)

Reply:
Maybe NS records can be used to send implicit NOTIFYs to downstream catalog consumers? Although explicit also-notifys are probably more useful in such scenarios.

Reply:
The text should definitely allow an implementation to use the NS records for that.

Reply:
I closed this conversation in favor of PR #53